packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -211,6 +262,20 @@ export class ApexReplayDebug extends LoggingDebugSession {
     this.printToDebugConsole(
       nls.localize('session_started_text', this.logContext.getLogFileName())
     );
+    // If the projectPath isn't set then don't bother with heap dump processing
+    if (this.projectPath) {
+      if (this.logContext.scanLogForHeapDumpLines()) {
+        if (
+          !await this.logContext.fetchOverlayResultsForApexHeapDumps(
+            this.projectPath
+          )
+        ) {
+          this.errorToDebugConsole(
+            nls.localize('heap_dump_error_wrap_up_text')
```

Comment:
If there's an error fetching heap dumps that's reported at the time it happens, this is a rollup error.